packages/syft/src/syft/store/mongo_document_store.py

```diff
@@ -293,6 +294,10 @@ def _set(
             if add_permissions is not None:
                 self.add_permissions(add_permissions)
 
+            if add_storage_permission:
+                # TODO: add storage permissions to Mongo store
```

Comment:
Would be great if we can do this before the weekend, if that doesnt work, lets merge without it and just test with single container stack nodes or dev nodes